server.js

```diff
@@ -110,7 +115,7 @@ app.post('/projects', function(req, resp) {
 			, website: req.body.website || ''
 			, picture: req.body.picture || ''
 			, statusCategory: determineStatusCategory(req.body.status) || ''
-			, coordinates: [coords.latitude, coords.longitude] || []
+			, coordinates: [(coords.latitude).toString(), (coords.longitude).toString()] || []
```

Comment:
not sure, everything in the database is in string format, changed it while getting the form working, might not be needed.